vignettes/GeospatialDataIntegration.Rmd

```diff
@@ -0,0 +1,647 @@
+    affiliation: EPA
+  - name: "Marc Weber"
+    affiliation: EPA
+  - name: "Dave Blodgett"
+    affiliation: USGS
+  - name: "Michael Dumelle"
+    affiliation: EPA
+  - name: "Shelly Thawley"
+    affiliation: EPA
+  - name: "Kenny Wong"
+    affiliation: ORISE at EPA
+---
+
+```{r knitsetup, include = FALSE}
+library(knitr)
+
+knitr::opts_chunk$set(
+  echo = TRUE,
+  warning = FALSE,
+  message = FALSE
+)
+```
+
+# Overview
+
+Let's walk through how to create an efficient and reproducible workflow
+that integrates several R Packages developed by the U.S. Environmental
+Protection Agency (EPA) and the U.S. Geological Survey (USGS) to support
+water quality programs (such as the Clean Water Act) and geospatial
+(watershed or waterbody) level analyses.
+
+This workflow demonstrates potential uses (beyond their original
+collection purpose) for publicly available water quality data from WQP.
+To start, participants will learn how to use EPA's Tools for Automated
+Data Analysis (TADA) R Package to retrieve, wrangle, harmonize, quality
+check, visualize and analyze WQP data from multiple organizations.
+
+Next, we will showcase how to bring in other web services and libraries
+for easy integration of additional hydrologic and geospatial data. We
+then plan to touch briefly on packages that can assist with building
+statistical models. Finally, we will demonstrate an example for
+analyzing water quality by Assessment Units (AUs), which are state or
+tribal nation defined watershed or waterbody areas used for CWA
+assessments and reporting water quality conditions to EPA and the
+public.
+
+**Intended Audience**
+
+Water Quality eXchange (WQX) and Water Quality Portal (WQP) community,
+Clean Water Act (CWA) community (EPA, States and Tribal Nations), water
+quality and geospatial data analysts/researchers, EPA/USGS and other
+federal agencies.
+
+**Leveraged R Packages**
+
+-   EPA: EPATADA, StreamCatTools, spsurvey, spmodel, SSN2
+
+-   USGS: dataRetrieval, nhdplusTools, hydroloom
+
+-   Fundamental geospatial packages: sf, prism, terra, leaflet and tmap
+
+# Install and load packages
+
+We will be leveraging the EPATADA R Package for WQP data retrieval,
+cleaning, visualization and other steps needed to prepare for analysis.
+Let's dive into Green Bay, WI!
+
+First, install and load the remotes package specifying the repo. This is
+needed before installing EPATADA because it is only available on GitHub
+(not CRAN).
+
+```{r install_remotes, results = 'hide'}
+install.packages("remotes",
+  repos = "http://cran.us.r-project.org"
+)
+library(remotes)
+```
+
+Next, install and load EPATADA using the remotes package. All other
+dependencies for this workflow will be downloaded automatically.
+
+```{r devsetup, include = F, eval = T}
+remotes::install_github("USEPA/EPATADA", 
+                        ref = "geospatial-vignette",
+                        dependencies = TRUE,
+                        force = TRUE)
+library(EPATADA)
+```
+
```

Comment:
Assume want to leave the ref=develop and get rid of the ref=geospatial-vignette when merging?